crypto/threads_win.c

```diff
@@ -143,6 +144,7 @@ CRYPTO_RCU_LOCK *ossl_rcu_lock_new(int num_writers, OSSL_LIB_CTX *ctx)
         return NULL;
 
     new->ctx = ctx;
+    new->rw_lock = CRYPTO_THREAD_lock_new();
```

Comment:
you need to check this for null in the if conditional below, or you'll have a leak

Reply:
I added the check.